core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -22,10 +22,12 @@ import scala.concurrent.duration._
 import scala.concurrent.Future
 import scala.util.Failure
 
-import com.redis.RedisClient
```

Comment:
Can we take the redis client dependency out now?

Reply:
yep; good catch. Done.